content/projects/oop/animals/part1/_index.md

```diff
@@ -125,6 +140,34 @@ home.makeAllSounds();
 //Dog barks
 
 
+```
+```
+# Python
+
+home = Home();
+dog1 = Dog();
```

Comment:
Creating an instance of these Dog and Cat classes, will expect that a name is also passed in, else it will cause an error